dataland-frontend/src/components/pages/SingleDataRequest.vue

```diff
@@ -58,6 +58,13 @@
                         :data-test="'datapoint-framework'"
                       />
                     </BasicFormSection>
+                    <BasicFormSection header="Information about company ownership">
+                      <p v-if="hasCompanyDataOwner">
+                        This company has a company owner. <br />
+                        The company owner will be informed about your data request.
```

Comment:
its possible for a company to have multiple owners. Consider changing to something like:
This company has at least one company owner.
The company owner(s) will be informed about your data request.

Reply:
done.